source/gui/settingsDialogs.py

```diff
@@ -724,7 +747,8 @@ def makeSettings(self, settingsSizer):
 		settingsSizerHelper = guiHelper.BoxSizerHelper(self, sizer=settingsSizer)
 		# Translators: A label for the synthesizer on the speech panel.
 		synthLabel = _("&Synthesizer")
-		synthGroup = guiHelper.BoxSizerHelper(self, sizer=wx.StaticBoxSizer(wx.StaticBox(self, label=synthLabel), wx.HORIZONTAL))
+		synthBox = wx.StaticBox(self, label=synthLabel)

```

Comment:
May I ask why this is changed? It seems this change hasn't been made to the BrailleSettingsPanel.

Reply:
Not a functional change, more of an incidental cleanup. I will make the same change in the BrailleSettingsPanel to ensure they are consistent.